containers/server-image/.env

```diff
@@ -0,0 +1,23 @@
+# MANAGER_USER=
```

Comment:
I think this file needs to be called the same way it was called in the original repo, or suma will not bootstrap correctly

Reply:
Ok, I thought it's required at runtime rather then the build, but I might be wrong here.

Reply:
this is requered just to configure the image after start up